scripts/check-dependencies.js

```diff
@@ -46,7 +46,8 @@ const assetsDir = path.join(__dirname, 'assets')
 const excludedPackages = [
     // The 'hello-world' app is explicitly designed to show what
     // you can do without our whole toolchain!
-    'hello-world'
+    'hello-world',
+    'hello-world-react'
```

Comment:
Is the comment valid for the 'hello-world-react' template? 
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/da18f29e92a3ba8a96bd5e42701ae1772faab5f9/scripts/check-dependencies.js#L47-L48

The `hello-world-react` template includes some dependencies that we also use in other packages of the monorepo.

My initial thought is to not exclude `hello-world-react` from the check-dependencies script. I think we preferably want to use the same dependency versions that we agree to use across all packages in the monorepo.  What is the reason to exclude `hello-world-react`?